src/Photos/Enums.cs

```diff
@@ -76,6 +76,11 @@ public enum PHVideoRequestOptionsDeliveryMode : long {
 	[Mac (10,12, onlyOn64 : true)]
 	[Native]
 	public enum PHCollectionListType : long {
+		[Introduced (PlatformName.iOS, 8, 0, message: "Will be removed in a future release")]
```

Comment:
The message only makes sense for the `[Deprecated]` attribute.
```suggestion
		[Introduced (PlatformName.iOS, 8, 0)]
```